src/main/resources/archetype-resources/src/main/webapp/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
+<html>
+<head>
+    <title>J2CL</title>
+    <meta content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no" name="viewport">
+
+    <link rel="stylesheet" href="css/${artifactId}.css">
```

Comment:
since i cant comment on the empty file, maybe put something in it?

Reply:
fixed